src/psfmachine/machine.py

```diff
@@ -323,12 +323,13 @@ def _create_delta_sparse_arrays(self, dist_lim=40, centroid_offset=[0, 0]):
 
     def _get_source_mask(
         self,
-        upper_radius_limit=28.0,
+        upper_radius_limit=36.0,
         lower_radius_limit=4.5,
         upper_flux_limit=2e5,
-        lower_flux_limit=100,
+        lower_flux_limit=40,
```

Comment:
this makes the rough mask more generous, good when scene motion is considerable, and allows for fainter pixels